### DISCUSSION TITLE
Can you use bluetooth proxies?

### DISCUSSION THREAD

**Discussion by DeWaHa** [dontinelli/fyta-custom_component#21] [Q&A] [answered]
Do you need the Fyta wifi hub or can you use Bluetooth proxies to integrate in home assistant?

**Comment by dontinelli** [comment, marked as answer]
The FYTA hub is needed, no direct access to the Beam data is possible.
Please note that according to my understanding the Beams only provide raw measurement data, that is then uploaded to the server via the hub. The data is processed on the FYTA server.
